services/cd-service/pkg/repository/repository.go

```diff
@@ -118,6 +123,16 @@ func openOrCreate(path string) (*git.Repository, error) {
 // Opens a repository. The repository is initialized and updated in the background.
 func New(ctx context.Context, cfg Config) (Repository, error) {
 	logger := logger.FromContext(ctx)
+
+	isMetricsEnabeld := ctx.Value("EnableMetrics")
+	if isMetricsEnabeld != nil && isMetricsEnabeld.(bool) {
+		var err error = nil
+		ddMetrics, err = statsd.New("127.0.0.1:8125", statsd.WithNamespace("Kuperbult"))
+		if err != nil {
+			logger.Fatal("datadog.metrics.error", zap.Error(err))
+		}
+	}
+
```

Comment:
the host/port `127.0.0.1:8125` should come from an environment variable. The value is good as a default though.

Reply:
Actually, I think if you put `nil`, statsd will read it from an env var, but I'm not sure about this.

Reply:
According to [DDStatsD](https://docs.datadoghq.com/developers/dogstatsd/?tab=hostagent#setup) DogStatsD is enabled by default over UDP port 8125 for Agent v6+, but it can be configured so I will add an env variable for it